UI/data/themes/Yami.qss

```diff
@@ -77,6 +77,7 @@ QWidget {
     selection-background-color: rgb(40,76,184);
     selection-color: palette(text);
     font-size: 10pt;
+    font-family: 'Open Sans', Helvetica, Arial, sans-serif
```

Comment:
I know Qt uses MS Shell Dlg as a sort of alias for "System font", though their docs don't seem to document it well. I would like to try placing it between Helvetica & Arial just in case.